normandy/classifier/tests/test_api.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+from normandy.base.utils import Whatever
+from normandy.recipes.tests import RecipeActionFactory, RecipeFactory
+
+@pytest.mark.django_db
+class TestFetchBundleAPI(object):
+
+    def test_it_works(self, client):
+        res = client.post('/api/v1/fetch_bundle/')
+        assert res.status_code == 200
+        assert res.data == {'recipes': []}
+
+    def test_it_serves_recipes(self, client):
+        recipe_action = RecipeActionFactory(arguments={'foo': 'bar'})
```

Comment:
Hmm. I think I'd prefer to assume that the default strategy is create, instead of putting `.create()` everywhere. I didn't realize the default used to be the build strategy, which is why I was confused.